homeassistant/components/risco/__init__.py

```diff
@@ -129,7 +129,7 @@ async def _async_setup_cloud_entry(hass: HomeAssistant, entry: ConfigEntry) -> b
     except CannotConnectError as error:
         raise ConfigEntryNotReady from error
     except UnauthorizedError as error:
-        _LOGGER.exception("Failed to login to Risco cloud")
+        _LOGGER.debug("Failed to login to Risco cloud %s", error)
```

Comment:
Alternatively we can pass the `error` as an argument to `ConfigEntryAuthFailed` and the error will be included in the warning that is logged by the config entries helper.

Reply:
This is already done because we're raising `ConfigEntryAuthFailed from error". I also confirmed that there's a warning in the logs containing the error message